src/foam/core/Currency.js

```diff
@@ -27,7 +27,7 @@
     {
       buildJavaClass: function(cls) {
         cls.extras.push(`
-          public String format(Long amount) {
+          public String format(long amount) {
```

Comment:
@jlhughes please let me know if you don't mind such changes ( you are the last person who worked on Currency according to git blame). thanks

Reply:
pretty mush the same

Reply:
java auto-boxing will convert, just less efficient.